plugins/inputs/intel_pmu/README.md

```diff
@@ -0,0 +1,196 @@
+# Intel Performance Monitoring Unit Plugin
+
+This input plugin exposes Intel PMU (Performance Monitoring Unit) metrics available through [Linux Perf](https://perf.wiki.kernel.org/index.php/Main_Page) subsystem.
+
+PMU metrics gives insight into performance and health of IA processor's internal components,
+including core and uncore units. With the number of cores increasing and processor topology getting more complex
+the insight into those metrics is vital to assure the best CPU performance and utilization.
+
+Performance counters are CPU hardware registers that count hardware events such as instructions executed, cache-misses suffered, or branches mispredicted.
+They form a basis for profiling applications to trace dynamic control flow and identify hotspots.
+
+### Configuration:
+```toml
+# Intel Performance Monitoring Unit plugin exposes Intel PMU metrics available through Linux Perf subsystem
+[[inputs.intel_pmu]]
+  ## List of filesystem locations of JSON files that contain PMU event definitions.
+  event_definitions = ["/var/cache/pmu/GenuineIntel-6-55-4-core.json", "/var/cache/pmu/GenuineIntel-6-55-4-uncore.json"]
+  
+  ## List of core events measurement entities.
+  [[inputs.intel_pmu.core_events]]
```

Comment:
Are users going to understand that there can be multiple entries in this list? TOML isn't the most straightforward config language. Maybe mention that you can have more than one of core_event sections. (same goes for uncore_event below.)

Reply:
Added proper mentions. Thanks!